### ISSUE TITLE
The “Expecting ‘,’ delimiter” error message is displayed when a parameter's value string contains quotation marks

### ISSUE THREAD

**Issue by SoftVision-SimonaRosu** [mozilla/experimenter#5339]
[Prerequisites]:
- Have Firefox Nightly 90.0a1 installed and opened.
- Have a new [Nimbus](https://stage.experimenter.nonprod.dataops.mozgcp.net/nimbus/) experiment created and opened.

[Steps to reproduce]:
1. Open the experiment's Branch section and click the "+Feature Configuration" button.
2. Open the Feature configuration dropdown menu and select the "privatebrowsing" feature.
3. Click the “ON” toggle and add the “{ "infoLinkText": "This “is” a link" }” configuration inside the “Value” textbox.
4. Click on the “Save” button from the bottom part of the page.
5. Observe the behavior.

[Expected result]:
 - The changes are saved and no error message is displayed.

[Actual result]:
 - The “Expecting ‘,’ delimiter” error message is displayed.

[Notes]:
- Attached a screenshot of the issue.
- [PI-1051]

![Nimbus_ErrorMessage](https://user-images.githubusercontent.com/50919288/120204915-03a17b80-c232-11eb-846f-e0567ce2b74e.png)

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/EXP-1365)
┆Issue Number: EXP-1365

**Comment by lmorchard** [comment]
Looks like the error is due to invalid JSON. We probably want to move that requirement to the review serializer and allow in-progress work to be saved even if invalid.

**Comment by SoftVision-SimonaRosu** [comment]
I have verified this issue using the latest Nightly 91.0a1 (Build ID: 20210704213917) on Windows 10 x64.
- This issue is no longer reproducible. It might have been fixed along the way with other changes that landed.
- The changes are saved and no error message is displayed when a parameter's value string contains quotation marks.
- Considering these I’m marking this issue as Verified Fixed.
